README.md

```diff
@@ -14,21 +25,24 @@ We recommend that you create a folder inside this project called `data` and put
 
 ### `.csv` knowledge base
 
-**fields:**
+The knowledge base has obligatory columns:
```

Comment:
I would change `obligatory` to `needed` because of the meaning of both words (this is not required, but just semantically looks better):

According to sir Google:
```
Obligatory
1. required by a legal, moral, or other rule; compulsory.
2. (of a ruling) having binding force.

Needed
1. require (something) because it is essential or very important

```

Reply:
good, these columns are needed and actually the load_csv raises an exception when some of them are not present
https://github.com/lgabs/dialog/blob/luan/update-docs-about-embeds/src/load_csv.py#L13